src/unity/lib/visualization/plot/plot.cpp

```diff
@@ -0,0 +1,58 @@
+#include <iostream>
+#include <memory>
+
+#include "plot.hpp"
+#include <unity/lib/visualization/process_wrapper.hpp>
+#include <unity/lib/visualization/thread.hpp>
+#include <unity/lib/visualization/transformation.hpp>
+#include <unity/lib/visualization/vega_data.hpp>
+#include <unity/lib/visualization/histogram.hpp>
+#include <unity/lib/visualization/item_frequency.hpp>
+#include <unity/lib/visualization/summary_view.hpp>
+
+namespace turi{
+  namespace visualization{
+    void Plot::show() {
+      using namespace turi;
```

Comment:
You don't need `using` for namespaces that you're already in. You can remove this and the next line.

Reply:
noted